wgsl/index.bs

```diff
@@ -995,6 +993,9 @@ statement
   | NOP SEMICOLON
   | assignment_stmt SEMICOLON
 
+return_stmt
```

Comment:
I don't think we need this duplicated here. Having it below only is fine.

Reply:
done